omod/src/main/webapp/resources/htmlforms/055b-HCTClientCard.xml

```diff
@@ -0,0 +1,596 @@
+<htmlform formName="HMIS 055B HIV Counseling and Testing Client Card"
```

Comment:
@dbaluku Why has the whole form changed?

Reply:
I think because it was a new PR with original files from another PR so there  was no trail of the files